src/aspire/utils/rotation.py

```diff
@@ -0,0 +1,289 @@
+"""
+Define a Rotation Class for customized rotation operations used by ASPIRE.
+"""
+
+import copy
+import numpy as np
+from numpy.linalg import norm
+from scipy.linalg import svd
+from scipy.spatial.transform import Rotation as sp_rot
+
+from aspire.utils import ensure
+from aspire.utils.matlab_compat import Random
+
+
+class Rotation:
+    """
+    Define a Customized Rotation class
+    """
+
+    def __init__(self, num_rots, seed=None, seq='ZYZ',
+                 angles=None, degrees=False, dtype=np.float32):
+        """
+        Initialize a Rotation object
+
+        :param num_rots: Total number of rotation sets
+        :param seed: Seed for initializing random rotations
```

Comment:
If None, then what?

Reply:
The random numbers are generated using the customized  Random class in aspire.utils.matlab_compat. If seed is None. it will continue from the previous random state. More notations were added in new commit 9f6e3b3.

Reply:
Cool..maybe say "If None, the random generator is not re-seeded."

Reply:
Done in new commit 9786c56.